jsapp/scss/components/_kobo.form-view.scss

```diff
@@ -880,6 +880,68 @@
   }
 }
 
+.form-modal__form.form-modal__form--data-attachment-columns {
+   color: darken($kobo-lightgray, 40%);
```

Comment:
I know that Ignacio will be adding quite some more grays to our palette. I wonder what should we do with that fact. This particular color is `hsl(230, 12%, 50%)`, which seems like best way would be to rename our grays to `$kobo-gray-50`. Let's add a new color like this and I will make a GH issue for renaming current ones

Reply:
Here it is https://github.com/kobotoolbox/kpi/issues/3235